scripts/database-subnet-whitelisting.sh

```diff
@@ -0,0 +1,33 @@
+set -e
+
+JSONPATH="[?(contains(type, 'Microsoft.DBforPostgreSQL/servers') && contains(name, 'perftest'))].name"
+POSTGRES_LIST=$(az resource list --query "${JSONPATH}" --subscription="" -o tsv | sort -u)
+
+for pg in ${POSTGRES_LIST[@]}
+do
+    case $pg in
+    
+    *"-idam"*)
```

Comment:
Is idam specific needed? can we not just query against *perftest* and build array?

Reply:
Also, can you swap to variable approach? Going forward we want it to be used in variable environments

Reply:
> Is idam specific needed? can we not just query against _perftest_ and build array?
 I saw the rg naming convention pattern (_-data_) for -idam-perftest postgres so had to include that conditional